standalone-metastore/pom.xml

```diff
@@ -292,6 +307,20 @@
       <version>${mockito-all.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
not sure about the licensing restrictions. Would be good to double check if you are not sure too. Also, why not have oracle jdbc through maven dependency too like others?

Reply:
See comments on DEV-README regarding not distributing any of these JDBC drivers.  I didn't include the Oracle one here because I couldn't find a maven package with it.  The only way I could find to get it was downloading it directly from Oracle's website, which requires accepting a license agreement.  If we can find a maven package somewhere we can use it.

Reply:
I see. Thanks for the clarification